libsolidity/formal/SMTChecker.cpp

```diff
@@ -75,6 +75,8 @@ void SMTChecker::analyze(SourceUnit const& _source, shared_ptr<Scanner> const& _
 	m_errorReporter.clear();
 }
 
+/** Visit functions. */
```

Comment:
doxygen has some grouping functionality: http://www.doxygen.nl/manual/grouping.html
Also please use `///` unless you are documenting a class.

Reply:
Changed to the recommended style